### PR TITLE
Adds normal wasps

### PR DESCRIPTION
**Pull request by chaosvolt**
- Adds a vermin version of wasps. Not sure if the small_bites flag really fits, we lack a sting flag. ID is "mon_wasp_small" because giant wasps already use the normal ID. Changing the giant wasps to use
  "mon_wasp_giant" would entail changes to source, tilesets, etc. Will go through with said changes if others insist.
- Added normal wasps to monstergroups (mainline and dinomod) in the same locations as giant wasps, using similar logic as non-giant insect spawns. Now if I knew a good way to have them sometimes be household pests...

### CODE REVIEW

**Comment by chaosvolt** [comment]
And thank you for the merge. ^^

File: data/json/monsters.json [hunk 1]
```diff
@@ -1342,6 +1342,35 @@
     "anger_triggers":["HURT", "FRIEND_DIED", "PLAYER_CLOSE"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_wasp_small",
+    "//": "id mon_wasp is already used, so changing would break tilests and other things",
+    "name": "wasp",
+    "species":"INSECT",
+    "default_faction":"wasp",
```
Comment (Coolthulhu) [inline data/json/monsters.json]: Faction "wasp" will probably make other creatures (zombies!) target the tiny wasp. Faction "vermin" should prevent it.

File: data/json/monsters.json [hunk 2]
```diff
@@ -1342,6 +1342,35 @@
...
+    "aggression":10,
+    "morale":20,
+    "speed":150,
+    "melee_skill":0,
+    "melee_dice":0,
+    "melee_dice_sides":0,
+    "melee_cut":0,
+    "dodge":3,
+    "armor_bash":2,
+    "armor_cut":4,
```
Comment (Coolthulhu) [inline data/json/monsters.json]: Armor?

**Comment by chaosvolt** [comment]
Doh. Vermin might make sense, yeah. @_@

And...armor was due to following the examples of dragonflies having the same armor value as giant ones. Seems that is an anomaly, as most of the game's bugs lack armor entirely.

**Comment by Midaychi** [comment]
@chaosvolt 
So I've been running into the vermin sized wasps lately. You can't target them normally and need a ranged or reach weapon, they're too small to hit with most reach weapons, they're very difficult to hit if at all with burst fire machineguns and the like, they keep attacking you ceaselessly (usually) to no effect, and I've even tried running one over with a car to no effect.

Can we at least make them not trigger the 'you see the wasp approaching' dialogue? I don't want to have to build laser sentry turrets and set off huge explosives just so I can stop a wasp from following me around and interrupting every attempt I make at doing anything that takes time.

**Comment by chaosvolt** [comment]
Probably should tweak their aggression and morale, I'm guessing. Or morale at least so you can run away from them without them doing a killer bee impression.

Because in real life, most of the times a wasp buzzes around me, it tends to fuck off after a while. >.<

**Pull request merged**